open_api_framework/conf/base.py

```diff
@@ -0,0 +1,662 @@
+# the code root.
+DJANGO_PROJECT_DIR = get_django_project_dir()
+BASE_DIR = os.path.abspath(
+    os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir)
+)
+
+#
+# Core Django settings
+#
+SITE_ID = config("SITE_ID", default=1)
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = config("SECRET_KEY")
+
+# NEVER run with DEBUG=True in production-like environments
+DEBUG = config("DEBUG", default=False)
+
+# = domains we're running on
+ALLOWED_HOSTS = config("ALLOWED_HOSTS", default="", split=True)
+USE_X_FORWARDED_HOST = config("USE_X_FORWARDED_HOST", default=False)
+
+IS_HTTPS = config("IS_HTTPS", default=not DEBUG)
+
+# Internationalization
+# https://docs.djangoproject.com/en/2.0/topics/i18n/
+
+LANGUAGE_CODE = "nl-nl"
+
+TIME_ZONE = "UTC"  # note: this *may* affect the output of DRF datetimes
+
+USE_I18N = True
+
+USE_L10N = True
+
+USE_TZ = True
+
+USE_THOUSAND_SEPARATOR = True
+
+#
+# DATABASE and CACHING setup
+#
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": config("DB_NAME", PROJECT_DIRNAME),
+        "USER": config("DB_USER", PROJECT_DIRNAME),
+        "PASSWORD": config("DB_PASSWORD", PROJECT_DIRNAME),
+        "HOST": config("DB_HOST", "localhost"),
+        "PORT": config("DB_PORT", 5432),
+    }
+}
+
+# keep the current schema for now and deal with migrating to BigAutoField later, see
+# https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-DEFAULT_AUTO_FIELD
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+CACHES = {
+    "default": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": f"redis://{config('CACHE_DEFAULT', 'localhost:6379/0')}",
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+            "IGNORE_EXCEPTIONS": True,
+        },
+    },
+    "axes": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": f"redis://{config('CACHE_AXES', 'localhost:6379/0')}",
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+            "IGNORE_EXCEPTIONS": True,
+        },
+    },
+    "oidc": {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": f"redis://{config('CACHE_DEFAULT', 'localhost:6379/0')}",
+        "OPTIONS": {
+            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+            "IGNORE_EXCEPTIONS": True,
+        },
+    },
+}
+
+#
+# APPLICATIONS enabled for this project
+#
+INSTALLED_APPS = [
+    # Note: contenttypes should be first, see Django ticket #10827
+    "django.contrib.contenttypes",
+    "django.contrib.auth",
+    "django.contrib.sessions",
+    # Note: If enabled, at least one Site object is required
+    "django.contrib.sites",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    # Optional applications.
+    "ordered_model",
+    "django_admin_index",
+    "django.contrib.admin",
+    # External applications.
+    "axes",
```

Comment:
Should the apps be ordered alphabetically within semantic groups? It would be neater, but we can live without it

Reply:
I could do this, but I'm not sure if order matters when it comes to these apps (`two_factor` and `maykin_2fa` for instance)

Reply:
Then let's leave it as it is